tests/func/test_pull.py

```diff
@@ -322,33 +313,13 @@ def test_pull_dataset_not_found_in_remote(
     assert str(exc_info.value) == "Error from server: Dataset not found"
 
 
-@pytest.mark.parametrize("cloud_type, version_aware", [("s3", False)], indirect=True)
-@skip_if_not_sqlite
-def test_pull_dataset_error_on_fetching_stats(
-    requests_mock,
-    cloud_test_catalog,
-    remote_dataset_info,
-):
-    requests_mock.get(
-        f"{STUDIO_URL}/api/datachain/datasets/stats",
-        status_code=400,
-        json={"message": "Internal error"},
-    )
-    catalog = cloud_test_catalog.catalog
-
-    with pytest.raises(DataChainError) as exc_info:
-        catalog.pull_dataset("ds://dogs@v1")
-    assert str(exc_info.value) == "Error from server: Internal error"
```

Comment:
what was that?

Reply:
I assume this test was for `error_on_fetching_stats` because the fixture was missing. We are no longer calling stats so we can delete the test.